src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -474,19 +474,18 @@ public MediathekArd(final FilmeSuchen ssearch, final int startPrio) {
     super(ssearch, SENDERNAME, 8, 50, startPrio);
   }
 
-
   private void addThema() {
     listeThemen.clear();
     MSStringBuilder seite = new MSStringBuilder(Const.STRING_BUFFER_START_BUFFER);
     meldungStart();
     final GetUrl getUrlIo = new GetUrl(getWartenSeiteLaden());
     seite = getUrlIo.getUri(SENDERNAME, ADRESSE_THEMA, StandardCharsets.UTF_8, 5 /* versuche */,
-        seite, "" /* Meldung */);
+            seite, "" /* Meldung */);
     if (seite.length() == 0) {
       Log.sysLog("ARD: Versuch 2");
       warten(2 * 60 /* Sekunden */);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)